### ISSUE TITLE
This svg format looks so nice

### ISSUE THREAD

**Issue by rockmandash** [prettier/prettier#3891]
![image](https://user-images.githubusercontent.com/7580792/35794612-60563e6c-0a91-11e8-91d4-ae0d74a8e748.png)

**Comment by azz** [comment]
Hi. Was there something wrong with the way Prettier is formatting svg code?  Please use the playground: https://prettier.io/playground/, and use the <kbd>Copy markdown</kbd> button.

**Comment by rockmandash** [comment]
There is nothing wrong.
I mean the code in path d=" HERE!!!! "
The image I show you, it split M, A, L etc... to different line.
This looks nice, but instead prettier format svg path d into one line.

**Comment by j-f1** [comment]
**Prettier 1.10.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAJgSwG4D4A6UABEagA4CGMAFkegLz4gCyAHAAwA0HhJf-JAIJEALAFZRYzkS4zpsgIwAmCcrG8BAgDJE1RDkQBaTIgDNMAGwuMQAcwBOcBCYD0BYqUo06N5koDM7PrsGpp8wuKS8tIK0URKAOyqKqFhRDqJEgbGIGaW1kyO6K7uqC5YeCCcIBBkMJjQAM7IoBT29hAA7gAKbQjNKBTYEJjoVSAUjTDIphQWjXDVAEb2FGAA1nAwAMqUYJhQtsgw9gCuiyDoEGAzcwvVBwv2MN2rtgC2FLfzFwBWjQAPABCqw2W22FHecC0Bzg33uNVOMDISKU8IulHsT2QICWFCWAE8LNBxmR7AcYAB1UY0ZAcapkiALSmrMg4slwJ7YOHVRwAR1OmEcrwoHy+SFmP2qC3emGOZwujQOtgscAAiqcIPB0dUYPjqehaUglLrVpZlQBhCDvT44qDQHkgU4LAAq+IGkoWAF8vUA)

**Input:**
```jsx
<div>
  <path d="M80,80
           A 45 45, 0, 0, 0, 125 125
           L 125 80 Z" fill="green" />
  <path d="M230 80
           A 45 45, 0, 1, 0, 275 125
           L 275 80 Z" fill="red" />
</div>
```

**Output:**
```jsx
<div>
  <path
    d="M80,80
           A 45 45, 0, 0, 0, 125 125
           L 125 80 Z"
    fill="green"
  />
  <path
    d="M230 80
           A 45 45, 0, 1, 0, 275 125
           L 275 80 Z"
    fill="red"
  />
</div>;

```

**Comment by lydell** [comment]
Considering how [Babel strips whitespace in JSX attribute values](http://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&code_lz=DwEwlgbgfAUABHYAHAhgFwBZxAXgEQCyAHAAwA0p8C1NcAgnACwCsTzZc5nHXAjAEysBzKrRoAZOMLik4ALTxwAZmAA2q_AHMATgFNdAO0UB6WAmTosuQvwDMJGSVFiEDFmx4denuPwDsQoLOLpL-rLIKymoaeHogJrDAxuDQQA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&lineWrap=true&presets=es2015%2Ces2016%2Ces2017%2Creact%2Cstage-0%2Cstage-1%2Cstage-2%2Cstage-3&prettier=false&targets=&version=6.26.0&envVersion=), I guess we could normalize that somehow. Not sure how though.

**Comment by j-f1** [comment]
suggestion:

```jsx
<div>
  <path
    d="M80,80
       A 45 45, 0, 0, 0, 125 125
       L 125 80 Z"
    fill="green"
  />
  <path
    d="M230 80
       A 45 45, 0, 1, 0, 275 125
       L 275 80 Z"
    fill="red"
  />
</div>;
```

**Comment by k15a** [comment]
@j-f1 This will be really hard with tabs and will result in tabs and spaces mixed.

**Comment by lydell** [comment]
I don't think we should try to align anything. Prettier doesn't align things in general. But I do think we could adjust the indentation or something.

**Comment by j-f1** [comment]
Oops, looks like we forgot to remove that label.

**Comment by peterlaxalt** [comment]
Bump. Please.

**Comment by fisker** [comment]
We don't even support `xml`, right?

**Comment by evilebottnawi** [comment]
Yes, it is feature
